### PR TITLE
test

### PR DESCRIPTION
**Pull request by lewisbirks**

### CODE REVIEW

File: ccd-definition/CaseEvent/Open.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
     "RetriesTimeoutURLSubmittedEvent": "1,2,3,4,5",
     "ShowSummary": "N",
     "ShowEventNotes": "N",
+    "CanSaveDraft": "Y",
```
Comment (lewisbirks) [inline ccd-definition/CaseEvent/Open.json:17]: test comment from intellij

Comment (lewisbirks) [inline ccd-definition/CaseEvent/Open.json:17]: test comment from intellij

Comment (akomisarek) [reply ccd-definition/CaseEvent/Open.json:17]: reply from intellij

Comment (akomisarek) [inline ccd-definition/CaseEvent/Open.json:17]: new comment from intellij

**Pull request closed without merging**